### PR TITLE
Minor: Add google tag manager changes for the pages

### PR DESCRIPTION
**Pull request by aniketkatkar97**

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -22,6 +23,15 @@ const DESCRIPTION =
 export type MyAppProps = MarkdocNextJsPageProps;
 
 export default function MyApp({ Component, pageProps }: AppProps<MyAppProps>) {
+  useEffect(() => {
+    window.dataLayer = window.dataLayer || [];
+    function updateGoogleTag() {
+      dataLayer.push(arguments);
+    }
+    updateGoogleTag("js", new Date());
+    updateGoogleTag("config", "G-2TW1XM6C89");
```
Comment (chirag-madlani) [inline pages/_app.tsx:32]: Can't we have everything in the script tag only?

Comment (aniketkatkar97) [reply pages/_app.tsx:32]: Actually doing that throws the error `dataLayer` is not defined. Thats why I moved it to useEffect

**Review by chirag-madlani — APPROVED** [review]

**Review by chirag-madlani — COMMENTED** [review]
Did we miss index.tsx?

**Comment by aniketkatkar97** [comment]
No we have the changes in index.tsx

**Comment by chirag-madlani** [comment]
@aniketkatkar97 pages/index.tsx

File: pages/[version]/[...slug].tsx [hunk 2]
```diff
@@ -80,6 +80,17 @@ export default function Article({
                 `,
         }}
       />
+      {/* Google tag (gtag.js) */}
+      <Script src="https://www.googletagmanager.com/gtag/js?id=G-2TW1XM6C89" />
```
Comment (Sachin-chaurasiya) [inline pages/[version]/[...slug].tsx:84]: @aniketkatkar97, can we create one common component for the google analytics script and use it in all the places?

This way if we want to change the script in the future we have to make the changes in one place only.

**Review by chirag-madlani — DISMISSED** [review]

File: components/GoogleAnalytics/GoogleAnalytics.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import Script from "next/script";
+
+function GoogleAnalytics() {
```
Comment (Sachin-chaurasiya) [inline components/GoogleAnalytics/GoogleAnalytics.tsx:3]:
```suggestion
function GoogleAnalyticsScript() {
```

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**